### PR TITLE
Feature/another data set

### PR DESCRIPTION
**Pull request by st3v3nhunt**
Further improvements needed:

* ~Reduce the unnecessary use of duplicated Promises~
* ~Test for the downloading of the file in a better/quicker way~

### CODE REVIEW

File: .istanbul.yml [hunk 1]
```diff
@@ -2,5 +2,5 @@ check:
   global:
     statements: 90
     lines: 90
-    branches: 90
-    functions: 80
+    branches: 65
```
Comment (st3v3nhunt) [inline .istanbul.yml:5]: Reduced as no longer using an externally hosted file and consequentially fewer branches are visited

File: app/lib/downloadFile.js [hunk 2]
```diff
@@ -1,30 +1,18 @@
...
-function scripts() {
-  const downloadLocation = constants.SCRIPTS.DOWNLOAD_LOCATION;
-  return downloadFile('SCRIPTS', downloadLocation);
-}
-
-module.exports = {
-  booking,
-  scripts,
-};
+module.exports = downloadFile;
```
Comment (markysoft) [inline app/lib/downloadFile.js]: Is the downloadFile.js still used?

**Review by markysoft — CHANGES_REQUESTED** [review]
Looks good, but I think the downloadFile.js is no longer used

File: app/lib/downloadFile.js (hunk 2 above)
Comment (st3v3nhunt) [reply app/lib/downloadFile.js]: Ah nope, I thought I'd deleted it.

**Comment by st3v3nhunt** [comment]
@markysoft File removed

**Review by markysoft — APPROVED** [review]

**Pull request merged**